tests/sentry/receivers/test_rules.py

```diff
@@ -0,0 +1,20 @@
+from sentry.analytics.events import project_created
+from tests.sentry.auth.test_access import AccessFactoryTestCase
+
+
+class DefaultRulesTest(AccessFactoryTestCase):
+    def test_include_all_projects_enabled(self):
```

Comment:
What is this test doing?

Reply:
I still need to write tests, it's just a placeholder for now